### ISSUE TITLE
Other languages support

### ISSUE THREAD

**Issue by renanmav** [orta/danger-plugin-spellcheck#27]
Hi guys,

First of all, congrats, that’s a great plugin.

I’m brazilian and we, brazilians, speak Portuguese. I tried to use this plugin on my codebase (which has a lot of Portuguese phrases) and danger was detecting it as miss spell words.

I saw that you use [cspell](https://github.com/streetsidesoftware/cspell) under the hood. Is that any support for other languages rather than English?

**Comment by orta** [comment]
I'd be open to a PR adding support for any language in https://github.com/streetsidesoftware/vscode-cspell-dict-extensions 👍🏻

**Comment by renanmav** [comment]
Sorry, but the PR would be addressed to this repo or [vscode-cspell-dict-extensions](https://github.com/streetsidesoftware/vscode-cspell-dict-extensions)?

**Comment by orta** [comment]
This repo would probably need an API so you can pass in one of those dictionaries

**Comment by renanmav** [comment]
I guess the API can be something like this:

```typescript
spellcheck({ dict: "pt_BR" })
```

And we would use [these dicts](https://github.com/streetsidesoftware/cspell-dicts).
